### PR TITLE
Self contained CLI 

### PR DESCRIPTION
**Pull request by knee-berts**

### CODE REVIEW

**Review by askmeegs — COMMENTED** [review]
Can we document the `init` command in the top level `README` before the `create` instructions? Thanks!

File: cli/pkg/cli_init/samples/default-config.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # These are the default values used for `gkekitctl create` when 
 # the user does not supply their own config.yml. Note that the user 
 # *does* have to provide their GCP project ID at runtime.  
-terraformState: local  
+terraformState: cloud
```
[the remark this answers is not in the document]
Comment (knee-berts) [reply cli/pkg/cli_init/samples/default-config.yaml:5]: I did. I think we should default to cloud storage.

**Comment by knee-berts** [comment]
> Can we document the `init` command in the top level `README` before the `create` instructions? Thanks!

All set.

File: README.md [hunk 2]
```diff
@@ -44,15 +44,12 @@ Sample inputs to modify the defaults can be found [here](cli/samples/).
 
 ## Install
 
-Clone the repo and move into the cloned directory:
+Set gkekitctl cli version type and OS then download the cli binary:
```
Comment (askmeegs) [inline README.md:47]: comma between `OS` and `then`

File: cli/pkg/config/config.go [hunk 3]
```diff
@@ -111,6 +115,15 @@ func InitConf(cfgFile string) *Config {
 		os.Exit(1)
 	}
 
+	// Set Tf Module Repo
+	err = setTfModuleRepo(conf.TFModuleRepo)
+	if err != nil {
+		log.Error(err)
```
Comment (askmeegs) [inline cli/pkg/config/config.go:121]: If `setTfModuleRepo` fails, do we want to exit fatal? right now we're just logging the error and continuing

File: README.md [hunk 4]
```diff
@@ -44,15 +44,12 @@ Sample inputs to modify the defaults can be found [here](cli/samples/).
...
 
 ```shell
-git clone https://github.com/GoogleCloudPlatform/gke-poc-toolkit.git && cd gke-poc-toolkit
-```
-
-Build the cli binary:
-```shell
-cd cli && go build .
+VERSION=$(curl -s https://api.github.com/repos/GoogleCloudPlatform/gke-poc-toolkit/releases/latest | grep browser_download_url | cut -d "/" -f 8 | tail -1)
+OS="darwin" # choice of darin or amd64
```
Comment (askmeegs) [inline README.md:51]: s/darin/darwin

File: cli/pkg/config/config.go (hunk 3 above)
Comment (knee-berts) [reply cli/pkg/config/config.go:121]: Added exit

**Pull request merged**